packages/page-settings/src/Metadata/NetworkSpecs.tsx

```diff
@@ -81,88 +81,170 @@ function NetworkSpecs ({ chainInfo, className }: Props): React.ReactElement<Prop
-          isDisabled
-          label={t<string>('Genesis Hash')}
-          value={networkSpecs.genesisHash}
-        />
-        <Input
-          className='full'
-          help={t<string>('Unit decides the name of 1 unit token, e.g. "DOT" for Polkadot')}
-          isDisabled
-          label={t<string>('Unit')}
-          value={networkSpecs.unit}
-        />
-        <Input
-          className='full'
-          help={t<string>('Prefix indicates the ss58 address format in this network, it is a number between 0 ~ 255 that describes the precise format of the bytes of the address')}
-          isDisabled
-          label={t<string>('Address Prefix')}
-          value={networkSpecs.prefix.toString()}
-        />
-        <Input
-          className='full'
-          help={t<string>('Decimals decides the smallest unit of the token, which is 1/10^decimals')}
-          isDisabled
-          label={t<string>('Decimals')}
-          value={networkSpecs.decimals.toString()}
-        />
-      </Columar.Column>
-      <Columar.Column>
-        {qrData.genesisHash && (
-          <QrNetworkSpecs
-            className='settings--networkSpecs-qr'
-            networkSpecs={debouncedQrData}
+        </td>
+      </tr>
+      <tr>
+        <td>
+
+          <Input
+            className='full'
+            help={t<string>('Unit decides the name of 1 unit token, e.g. "DOT" for Polkadot')}
+            isDisabled
+            label={t<string>('Unit')}
+            value={networkSpecs.unit}
           />
-        )}
-      </Columar.Column>
-    </Columar>
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          <Input
+            className='full'
+            help={t<string>('Prefix indicates the ss58 address format in this network, it is a number between 0 ~ 255 that describes the precise format of the bytes of the address')}
+            isDisabled
+            label={t<string>('Address Prefix')}
+            value={networkSpecs.prefix.toString()}
+          />
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <Input
+            className='full'
+            help={t<string>('Decimals decides the smallest unit of the token, which is 1/10^decimals')}
+            isDisabled
+            label={t<string>('Decimals')}
+            value={networkSpecs.decimals.toString()}
+          />
+        </td>
+      </tr>
+    </Table>
   );
 }
 
 export default React.memo(styled(NetworkSpecs)`
-  top: .3rem;
 
-  .settings--networkSpecs-colorButton {
-    display: flex;
-    flex-direction: row;
+  td {
+      padding: 0;
+
+      .input.ui--Input input {
+          border: none !important;
+          background: transparent;
+      }
+  }
+
+  .settings--networkSpecs-name {
+    position: relative;
+        .settings--networkSpecs-logo {
+          height: 32px;
+          left: 12px;
+          position: absolute;
+          top: 1rem;
+          width: 32px;
+        }
```

Comment:
The spacing here and in the remainder is weird - just ensure it is spaces, 2 indent. So you would end up with -

```js
.settings--networkSpecs-name {
    position: relative;
    
    .settings--networkSpecs-logo {
      height: 32px;
      ...
```